docs/gpio.md

```diff
@@ -0,0 +1,215 @@
+# GPIO
+
+The GPIO driver on current version supports:<a href="https://github.com/matrix-io/matrix-creator-malos/blob/av/doc_gpio/docs/gpio_diagram.jpg"><img src="https://github.com/matrix-io/matrix-creator-malos/blob/av/doc_gpio/docs/gpio_diagram.jpg" align="right" width="420" ></a>
+
+* GPIO pin input
+* GPIO pin output
+* GPIO updates (state from all pins)
+
+The driver follows the [MALOS protocol](../README.md#protocol).
+
+### GPIO basic electronic specs
+
+* voltage 3.3v
```

Comment:
According to FPGA DS: Voltage DC 0.60 to 4.10 V 20% overshoot duration –0.75 to 4.25 V 8% overshoot duration(5) –0.75 to 4.40 V